docs/sdkx-react-native/release-notes-react-native.mdx

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 290
+title: Release Notes
+description: Read the Release Notes in detail
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Release Notes
+
+## 10.2.1-beta.5
+
+_Feb 10th, 2023_
```

Comment:
Need to add release notes for latest release.

Reply:
Updated.